beater/config_test.go

```diff
@@ -40,6 +40,7 @@ func TestConfig(t *testing.T) {
 			config: []byte(`{
         "host": "localhost:3000",
         "max_header_size": 8,
+		"idle_timeout": 4s,
```

Comment:
odd that this whitespace didn't cause intake issues

Reply:
Is the idle_timeout configurable? If not the the ability to set a large idle_timeout (previously same read_timeout) is lost and therefore you can't have an uninterrupted TCP stream. This is a serious feature loss imho as HTTP over an uninterrupted TCP connection is the only thing that can make a hard guarantee not to cause the race condition in #1936. In earlier versions I could at lest work around it with configuration even if defaults were problematic.